students/AndrewMiotke/session06/walnut_party/walnut_party.py

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python3
+
+"""
+When squirrels get together for a party, they like to have walnuts.
+A squirrel party is successful when the number of walnuts is between
+40 and 60, inclusive. Unless it is the weekend, in which case there
+is no upper bound on the number of walnuts.
+
+Return True if the party with the given values is successful,
+or False otherwise.
+"""
+
+
+def walnut_party(walnuts, is_weekend):
+    if walnuts <= 30 and is_weekend == False:
```

Comment:
This can be done much neater if you only apply the logic to return true and return false all other times. Ex:
if weekend and walnuts >= 40:
return true #handle weekend true case
if not weekend and walnuts >= 40 and walnuts <= 60:
return true #handles not weekend true case

return false #this will return false if our two options are not true